pysb/export/sbml.py

```diff
@@ -90,84 +70,211 @@ class SbmlExporter(Exporter):
     Inherits from :py:class:`pysb.export.Exporter`, which implements
     basic functionality for all exporters.
     """
+    def __init__(self, *args, **kwargs):
+        super(SbmlExporter, self).__init__(*args, **kwargs)
+        if not libsbml:
```

Comment:
Suggest moving the libsbml check even before the super class init call to indicate it's absolutely required before we proceed.